app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+API_KEY_PARAM = "key"
+CITY_PARAM = "q"
+CITY = "Paris"
```

Comment:
According to the checklist, constants should be used for the base URL and parameters. You've done well with the `BASE_URL`, `API_KEY_PARAM`, and `CITY_PARAM` constants. But the `CITY` variable should also be a constant, hence its name should be in uppercase, i.e., `CITY = 'Paris'` should be changed to `CITY = 'PARIS'`.

Reply:
I should have 'Paris' in my output